### ISSUE TITLE
Rogue - Room information is getting gagged frequently when stacking direciton & bs/cr mob

### ISSUE THREAD

**Issue by Gyrx** [Nyyrazzilyss/NyyLIB#5]
Script version: 011dev 7/6/2017

Log:
< 738h/910H 130v/130V P: std > 
n;bs 2.orc
A Guardpost at the Foot of the Staircase
   This building, if it can be called a building, is nothing more
than a cube of rusty metal panels than have been riveted together
very haphazardly.  In addition to the opening for the door leading
back out towards the stairwell, a small window has been cut
into the metal.  A pair of human-sized wooden chairs are set
in the back corners of the room, the only visible creature comfort
in what would otherwise be an empty metal box.  A lantern
has been hung from a rivet on the back wall, providing meager light.
< 740h/910H 129v/130V P: std >

A gruff orc sentry makes a strange sound, as you place a glittering dagger of the Nurturing 
Matriarch in his back.
A gruff orc sentry makes a strange sound, as you place a red dragon fanged dagger in his back.
A gruff orc sentry sends you sprawling.

< 740h/910H 129v/130V T: Ezdad TC: small wounds E: sentry EC: nasty wounds EP: std P: sit > 

----------------------
2nd, slightly different scenario that had the same impact when stacking commands:

< 1037h/910H 130v/130V T: Ezdad TC: excellent E: sentry EC: pretty hurt EP: std P: std >  (x2)
escape n;appp;s;bs 2.orc
At the Foot of a Metal Staircase
   The metal staircase abruptly ends here on a small slab of stone.
Streaks of red run through the gray rock, and the surface has the 
appearance of a tooled surface rather than a natural formation.
Four torches on metal poles rest at each corner of the landing,
throwing shadows that dance across the earthen walls of the room.
A slab of the same streaked rock has been rock has been shaped
into a rectangular door set into a thick wooden frame on the east
wall of the room.  Two rusting metal shacks rest at the north
and south ends of the room, and the metal staircase spirals up
< 1037h/910H 129v/130V P: std >

You get a vial of black liquid from a portable hole.

< 1037h/910H 129v/130V P: std >

You can't carry any more.

< 1037h/910H 129v/130V P: std >

You generously apply poison to a glittering dagger of the Nurturing Matriarch.

< 1037h/910H 129v/130V P: std >

You generously apply poison to a red dragon fanged dagger.

< 1037h/910H 129v/130V P: std >

A Guardpost at the Foot of the Staircase
   This building, if it can be called a building, is nothing more
than a cube of rusty metal panels than have been riveted together
very haphazardly.  In addition to the opening for the door leading
back out towards the stairwell, a small window has been cut
into the metal.  A pair of human-sized wooden chairs are set
in the back corners of the room, the only visible creature comfort
in what would otherwise be an empty metal box.  A lantern
has been hung from a rivet on the back wall, providing meager light.
The only apparent exit is back to the north.
Exits: - North         [15720] 
A gruff-looking orc dutifully stands at his post. (x3)

< 1037h/910H 128v/130V P: std >

A gruff orc sentry makes a strange sound, as you pl

**Comment by Nyyrazzilyss** [comment]
The dev update posted 7/10 has taken steps to correct this. If still occuring (with that version) add an update on this thread.

**Comment by Gyrx** [comment]
Updated to 7/10 Dev.

Observing an improvement but some issues when escaping north.

Log:


< 772h/865H 73v/130V T: Ezdad TC: small wounds E: rowan EC: few wounds EP: std P: sit > 
escape n

Grrrrrrrrrrrr... a large rowan tree growls at you.
You dodge a large rowan tree's vicious attack.

< 772h/865H 74v/130V T: Ezdad TC: small wounds E: rowan EC: few wounds EP: std P: sit > 

Grrrrrrrrrrrr... a large rowan tree growls at you.
a large rowan tree misses you with its bludgeon.

< 772h/865H 75v/130V T: Ezdad TC: small wounds E: rowan EC: few wounds EP: std P: sit > 
You dodge a large rowan tree's vicious attack. (x2)
a large rowan tree severely wounds you with its massive bludgeon.
OUCH!  That really did HURT!
a large rowan tree severely wounds you with its massive bludgeon.
OUCH!  That really did HURT!
You dodge a large rowan tree's vicious attack.
a large rowan tree misses you with its bludgeon.
a large rowan tree barely wounds you with its weak bludgeon.
a large rowan tree misses you with its bludgeon.
a large rowan tree barely wounds you with its weak bludgeon. (x2)
a large rowan tree misses you with its bludgeon. (x3)
You dodge a large rowan tree's vicious attack.
a large rowan tree misses you with its bludgeon.
a large rowan tree barely wounds you with its weak bludgeon.
a large rowan tree wounds you harshly with its massive bludgeon.
OUCH!  That really did HURT!
a large rowan tree misses you with its bludgeon.
a large rowan tree parries your futile lunge at it.
Your massive pierce barely wounds a large rowan tree.
Your attack glances off a large rowan tree's shield, but finds its mark!
Your powerful pierce barely wounds a large rowan tree.
Your massive pierce barely wounds a large rowan tree.

< 524h/865H 76v/130V T: Ezdad TC: few wounds E: rowan EC: few wounds EP: std P: sit > 
< 524h/865H 77v/130V T: Ezdad TC: few wounds E: rowan EC: few wounds EP: std P: sit >

A Ring of Rowan Trees
Exits: - North         [7691] 
Blood splotches cover everything in the area.
A large pool of water which glints silver fills the area.
The faerie sage, Tatterfoal sits here musing the meaning of life.
a large rowan tree stands here, fighting YOU! (x2)
A large rowan tree stands sentinel here.
a large rowan tree stands here, fighting YOU! (x4)

< 524h/865H 78v/130V T: Ezdad TC: few wounds E: rowan EC: few wounds EP: std P: sit >

You clamber to your feet.

< 524h/865H 78v/130V T: Ezdad TC: few wounds E: rowan EC: few wounds EP: std P: std >

South Side of the Hill
   The traveling is easy down the south side of the hill.  There
really is no path here, just a grassy slope dotted with wildflowers.
To the north is the top of the hill, to the south is what looks to be
< 524h/865H 77v/130V P: std > 

   680/680   HP  106/106  MV   P  + Munintokik

< 524h/865H 77v/130V P: std >

Why?  You are not fighting anyone.

< 524h/865H 78v/130V P: std >

South Side of the Hill
Exits: - North - South  [7690] 
You sense a hidden life form in the room.
A fire elemental is here blazing with heat!
Munintokik  (Duergar) sits resting in mid-air.

< 524h/865H 78v/130V P: std >

**Comment by Gyrx** [comment]
Here's a log from 7/15 dev.

Brief Mode is set to OFF

Scenario:
1. In combat
2. manually send to mud "escape e;w;cr priest"
3. Observe that when I reenter  a) part of the room description is gagged b) the room exit directions, people, and any items in the room were gagged c) the follower (Zissar) following me back into the room was also gagged so I did not know if they stayed behind or came with me


Log:
< 854h/440H 130v/130V T: Dagk TC: few scratches E: priest EC: pretty hurt EP: std P: std > 
[Attacking with circle]
Dagk sends a kindly priest sprawling with a powerful bash.

< 854h/440H 130v/130V T: Dagk TC: few scratches E: priest EC: pretty hurt EP: sit P: std >

You circle around a kindly priest stabbing at his back.
escape e;w;cr priest

< 853h/440H 130v/130V T: Dagk TC: few scratches E: priest EC: pretty hurt EP: sit P: std > 

Dagk parries a kindly priest's lunge at him.
Dagk barely wounds a kindly priest with his awesome slash.
Dagk badly wounds a kindly priest with his massive slash.
Dagk barely wounds a kindly priest with his mighty slash.
Your massive pierce badly wounds a kindly priest. (x4)
Your mighty pierce slightly wounds a kindly priest.

< 844h/440H 130v/130V T: Dagk TC: few scratches E: priest EC: awful EP: sit P: std > 
Entry Hall of the Keep's Cathedral
Exits: - East  - West   [87313] 
A pool of blood covers the ground.
Corpse of a kindly priest is lying here.
Dagk Slash Gut (Troll) stands here, fighting a kindly priest.
A kindly priest sits here, fighting Dagk.

< 837h/440H 130v/130V T: Dagk TC: few scratches E: priest EC: awful EP: sit P: std >

Center Aisle in the Cathedral
[Attacking with circle]
   A long carpet of purple and blue heads forward from the entry hall
up to an altar.  Flanking each side of the this aisle are many rows
of pews, each uniquely carved from a piece of dark wood.  Large
windows rise high above the pews, each pane containing a collage of
stained colors.  Around the windows of brilliant glass golden figurines
along with ropes of silver form a spectacular frame.   It is said, that
during certain times of the day, the light which filters through these
windows lights up the cathedral with a multitude of dazzling rainbows.
Exits: - North - East  - South - West                  [87313] 
Zissar  (Yuan-Ti) stands in mid-air here.
You escape eastward!

< 836h/440H 129v/130V P: std >

Entry Hall of the Keep's Cathedral
   Fine woven tapestries hang freely upon the cold stone walls of
the cathedral.  Each of the intricate specimens contains a scene
of some religious significance.  Just below, resting upon a table
of three legs, are a multitude of burning candles.  The light shed
by the slender wax is enough to illuminate the hall.  A small statue
is here, cast in granite, it is of a figure holding out a bowl.  Next
to the statue is a box, made of iron, which is firmly secured to
< 835h/440H 128v/130V P: std >

You circle around a kindly priest stabbing at his back.

< 835h/440H 128v/130V T: Dagk TC: few scratches E: priest EC: awful EP: sit P: std > 
Your massive pierce severely wounds a kindly priest. (x2)
Your massive pierce wounds a kindly priest harshly.
A kindly priest staggers from your massive pierce!
A kindly priest misses Dagk with his hit.
Dagk severely wounds a kindly priest with his mighty slash.
Dagk wounds a kindly priest harshly with his mighty slash.
Dagk enshrouds a kindly priest in a mist of blood with his massive slash!

< 834h/440H 128v/130V T: Dagk TC: few scratches E: priest EC: awful EP: sit P: std >

**Comment by Gyrx** [comment]
On my invoker while stacking a move comment and FM I was unexpectedly seeing the Room name.  Unexpected room name is in bold.

I'm not sure if this is related here or should be a new issue:

Log:
scan

You scan east...
A Mist Enshrouded Cemetery
Room size: Large (L:40 ft  W:40 ft  H:20 ft)
Exits: -E -S -W
A translucent, robed figure drifts about.

You scan south...
A Mist Enshrouded Cemetery
Room size: Large (L:40 ft  W:40 ft  H:20 ft)
Exits: -E -W

You scan west...
A Mist Enshrouded Cemetery
Room size: Large (L:40 ft  W:40 ft  H:20 ft)
Exits: -N -E

< 174h/222H 113v/128V P: std > 
e;fm robed
A Mist Enshrouded Cemetery
Room size: Large (L:40 ft  W:40 ft  H:20 ft)
Exits: -E -S -W        [47714] 
A translucent, robed figure drifts about.

< 176h/222H 115v/128V P: std >

You start chanting... [Casting: fm robed (Force Missiles)]

< 176h/222H 115v/128V P: std > 

Casting: Force Missiles **

< 177h/222H 116v/128V P: std > 

Casting: Force Missiles *

< 177h/222H 117v/128V P: std > 

You complete your spell...[Completed spell: fm robed (Force Missiles)]
You send a powerful force missile slamming into a Zhentarim spirit! (x3)

< 178h/222H 117v/128V P: std > 
A Zhentarim spirit stops chanting abruptly!

< 178h/222H 117v/128V P: std > 
fm robed

A Zhentarim spirit severely wounds you with his strong beat.
A Zhentarim spirit starts casting a spell.

< 159h/222H 118v/128V T: Korris TC: few wounds E: zhentarim EC: pretty hurt EP: std P: std > 
**A Mist Enshrouded Cemetery**
< 159h/222H 119v/128V T: Korris TC: few wounds E: zhentarim EC: pretty hurt EP: std P: std >

You start chanting... [Casting: fm robed (Force Missiles)]

< 159h/222H 119v/128V T: Korris TC: few wounds E: zhentarim EC: pretty hurt EP: std P: std > 

Casting: Force Missiles **

< 160h/222H 120v/128V T: Korris TC: few wounds E: zhentarim EC: pretty hurt EP: std P: std > 

You complete your spell...[Completed spell: fm robed (Force Missiles)]
You send a powerful force missile slamming into a Zhentarim spirit!
A Zhentarim spirit is mortally wounded, and will die soon, if not aided.
A Zhentarim spirit slumps to the ground.
Your last force missile rips into a Zhentarim spirit and tears him in two!
a Zhentarim spirit is dead! R.I.P.
A Zhentarim spirit slumps to the ground.
You receive 2,212 XP (0.01%) experience.
Your blood freezes as you hear the rattling death cry of a Zhentarim spirit.
A Zhentarim spirit stops chanting abruptly!

< 160h/222H 122v/128V P: std > 
search corpse
**A Mist Enshrouded Cemetery**
< 160h/222H 123v/128V P: std >

You don't find anything.

< 160h/222H 123v/128V P: std > 
search corpse
You search exhaustively and conclude there is nothing to be found!

< 160h/222H 125v/128V P: std >

**Comment by Hibbidy** [comment]
I noticed it again yesterday when i had to hide after circling a dragon due to aggros around -
So I just tested it hiding and non-hiding - at least below, seemed to happen when hiding after escaping - then gags the room on next entry -

< 734h/597H 174v/174V T: Nonog TC: excellent E: bella EC: excellent EP: std P: std > 

Bella Earthtoes attacks you, landing a single hit.
Bella Earthtoes starts casting an offensive spell.
Your wonderful glittering dagger glows brightly as your body starts to flicker, in
and out of visibility! You spring to the side, dodging Bella Earthtoes's attack.
the holy blessing of Yondalla has taken hold upon you! Bella Earthtoes cries
out as you plunge your dagger perfectly into the center of her back. You grin
as your dagger draws the very lifeforce out of her.
You attack Bella Earthtoes, landing five hits.

< 870h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few scratches EP: std P: std > 
esc e

Bella Earthtoes glows as she draws on the power of Yondalla.

< 867h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few scratches EP: std P: std > 
h
hide
save

You attack Bella Earthtoes, landing four hits.

< 864h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

Bella Earthtoes is a small-sized level 56 halfling. Are you mad!?

< 864h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

Your massive pierce barely wounds Bella Earthtoes.

< 863h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

< 863h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

Nyyrazzilyss ASSOC:: 'Sucks that I can add something and have it work just fine on some classes, but 
other classes break :('
At the End of Sunset Lane
Exits: -N -E -S -W     [48930] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.
You escape eastward!

< 862h/597H 173v/174V P: std >

You attempt to hide yourself.

w;cr halfling
< 862h/597H 173v/174V P: std > 
Saving...

< 851h/597H 174v/174V P: std > 
< 851h/597H 173v/174V P: std >

You stop trying to hide.
Your massive pierce barely wounds Bella Earthtoes.

< 850h/597H 173v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std > 
Bella Earthtoes glows as she draws on the power of Yondalla.

< 850h/597H 173v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std > 

You score a CRITICAL HIT!
You attack Bella Earthtoes, landing five hits.

< 845h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std > 
esc e
< 840h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

Bella Earthtoes completes her spell...
Bella Earthtoes utters the words, 'yjrr pzar'.
Bella Earthtoes summons a holy light and heals her own wounds.
Bella Earthtoes is a small-sized level 56 halfling. Are you mad!?

< 840h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

A Small Shrine to the Halflings Gods
Exits: -E              [48930] 
A pool of blood covers the ground.
Bella Earthtoes stands here, fighting YOU!

< 839h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: std P: std >

At the End of Sunset Lane
Exits: -N -E -S -W     [48930] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.
You escape eastward!

< 839h/597H 173v/174V P: std > 
w;cr halfling
A Small Shrine to the Halflings Gods
Exits: -E              [48930] 
A pool of blood covers the ground.
A robust halfling kneels here reverently.

< 834h/597H 173v/174V P: std > 
You circle around Bella Earthtoes stabbing at her back.

< 833h/597H 173v/174V E: bella EC: small wounds EP: std P: std > 
esc e

Severely weakened, Bella Earthtoes falls to her knees!
You attack Bella Earthtoes, landing four hits.

< 829h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: knl P: std > 
Bella Earthtoes stops chanting abruptly!

< 829h/597H 174v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: knl P: std > 
w;cr halfling
At the End of Sunset Lane
Exits: -N -E -S -W     [48928] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.
You escape eastward!

< 825h/597H 173v/174V P: std >

At the End of Sunset Lane
Exits: -N -E -S -W     [48929] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.

< 825h/597H 173v/174V P: std >

A Small Shrine to the Halflings Gods
Exits: -E              [48929] 
A pool of blood covers the ground.
Bella Earthtoes crouches here.

< 825h/597H 172v/174V P: std >

Your massive pierce barely wounds Bella Earthtoes.

< 824h/597H 172v/174V T: Nonog TC: excellent E: bella EC: small wounds EP: knl P: std > 

Bella Earthtoes attacks you, landing a single hit.
You attack Bella Earthtoes, landing two hits.

< 762h/597H 173v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: knl P: std > 

Bella Earthtoes attacks you, landing a single hit.
Bella Earthtoes rises to her feet.
You attack Bella Earthtoes, landing three hits.

< 708h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std > 
esc e;h
hide
save
Bella Earthtoes is a small-sized level 56 halfling. Are you mad!?

< 707h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std > 
< 707h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std >

At the End of Sunset Lane
Exits: -N -E -S -W     [48929] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.
You escape eastward!

< 707h/597H 173v/174V P: std >

You attempt to hide yourself.

< 707h/597H 173v/174V P: std > 
w;cr halfling
Saving...

< 703h/597H 174v/174V P: std > 
< 703h/597H 173v/174V P: std >

You stop trying to hide.

Your massive pierce barely wounds Bella Earthtoes.

< 702h/597H 173v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std > 

Bella Earthtoes completes her spell...
Bella Earthtoes utters the words, 'yjrr pzar'.
Bella Earthtoes summons a holy light and heals her own wounds.
You attack Bella Earthtoes, landing four hits.

< 701h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std > 
esc e
< 699h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std >

Bella Earthtoes attacks you, landing a single hit.
Bella Earthtoes starts casting an offensive spell.
You attack Bella Earthtoes, landing five hits.
A Small Shrine to the Halflings Gods
Exits: -E              [48930] 
A pool of blood covers the ground.
Bella Earthtoes stands here, fighting YOU!

< 629h/597H 174v/174V T: Nonog TC: excellent E: bella EC: few wounds EP: std P: std > 
At the End of Sunset Lane
Exits: -N -E -S -W     [48930] 
A large boar stands here snarling.
A dwarven adventurer moves through Silverymoon.
You escape eastward!
